std_msgs/Header header
apriltag_toolbox/AprilTagDetection[] detections